docs/releases/saros-e_15.0.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Saros Release Notes
+---
+
+The Saros Team is happy to announce Saros for Eclipse 15.0.0 and Saros for IntelliJ 0.1.0.
```

Comment:
Also remove mention of Saros/I?

Reply:
Ah, sorry I forgot this comment. I will remove this.